docs/language/overview.md

```diff
@@ -105,14 +105,14 @@ All available operators are listed on the [reference page](operators/README.md).
 ### 2.1 Dataflow Sources
 
 In addition to the data sources specified as files on the `zq` command line,
-a source may also be specified with the [from operator](operators/from.md#operator).
+a source may also be specified with the [`from` operator](operators/from.md).
```

Comment:
This was a convention I pre-sold with @nwt. In many places, this doc was already saying "\<some> operator", "\<some> function", etc., which I think does a nice job of helping the user start to understand the differences between operators, simple functions, and aggregate functions, and also disambiguating in cases where there's one of both flavors (e.g., `fuse`). However, we also agreed that it would be good practice to start putting the operator name in fixed width font, so I've started doing that here. The other thing I've adopted as a convention is dropping the extra `#operator` that was at the end, which is how the function links were already written.